torch/_prims_common/wrappers.py

```diff
@@ -264,6 +265,7 @@ def _fn(*args, out=None, **kwargs):
                         # These two operations are done in-place
                         _maybe_resize_out(o, r.shape)
                         _safe_copy_out(copy_from=r, copy_to=o, exact_dtype=exact_dtype)  # type: ignore[arg-type]
+                        torch._C._set_conj(o, r.is_conj())
```

Comment:
It's important this uses inplace `_set_conj` and not `conj` + reassign because there are checks in core that verify whether the tensor is the same.